packages/blockchain-link-utils/src/solana.ts

```diff
@@ -29,10 +40,28 @@ type TransactionEffect = {
 };
 
 export function getTransactionEffects(transaction: ParsedTransactionWithMeta): TransactionEffect[] {
+    // TODO(vl): after token PR is merged, get WSOL mint from there
+    const wSolTransferInfo = transaction.transaction.message.instructions?.find(
```

Comment:
On second thought I'll add an explanatory comment as to why we need to do this at all, i.e. why WSOL sending is weird.

Reply:
Yes definitely, I would put this wsol logic into some helper, like `isWsolTransfer` and add comment in place where the helper is used.